gitbook-docs/data_model.md

```diff
@@ -201,7 +202,11 @@ The `updates` holds an array (JSON array) of updates, each of which has a `sourc
 An `update` has a single `source` value and it applies to each of the `values` items.
 In cases where you can get data from only a single source the source may be omitted and the receiver may fill it in when multiplexing data from several sources.
 
-Each 'value' item is then simply a pair of 'relative path', and 'value'.
+Each `value` item is then simply a pair of `path` and `value`.
+The `path` must be a _leaf path_: it must be a path to a leaf the of the full model.
+A leaf is where the actual value of the Signal K property is and where `timestamp`, `$source` and `values` properties are in the full model.
+The value is often a scalar - a numeric value, as in the example above, but it can also be an object.
```

Comment:
In the full model a leaf node is _always_ an object - it needs to be to store the `timestamp` and other meta information about the value. The scalar quantity is stored in a `value` property of that object. However, for `object` values we [mix](https://github.com/SignalK/specification/blob/master/schemas/groups/navigation.json#L396) the commonValueFields into the leaf object which makes this inconsistent.

Perhaps the solution for that is to modify the full model to always use the `value` property for storing the value and stop mixing in.